<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoQuestList</class>
 <widget class="QDialog" name="GeoQuestList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>List of questions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Question1Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question2Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question3Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question4Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question5Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question6Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question7Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question8Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question9Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question2Btn10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResultBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>List of questions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
